include/mbedtls/config_psa.h

```diff
@@ -220,6 +220,65 @@ extern "C" {
 #endif /* !MBEDTLS_PSA_ACCEL_KEY_TYPE_RSA_PUBLIC_KEY */
 #endif /* PSA_WANT_KEY_TYPE_RSA_PUBLIC_KEY */
 
+#if defined(PSA_WANT_KEY_TYPE_AES)
+#if !defined(MBEDTLS_PSA_ACCEL_KEY_TYPE_AES)
+#define MBEDTLS_PSA_BUILTIN_KEY_TYPE_AES 1
+#define MBEDTLS_AES_C
```

Comment:
MBEDTLS_AES_C is still used in psa_crypto.c. and should be changed. It should be changed to PSA_WANT_KEY_TYPE_AES (and not MBEDTLS_PSA_BUILTIN_KEY_TYPE_AES) as the code it guards is, I think, also necessary in the case of an AES accelerator. Same comments applies to MBEDTLS_ARC4_C, MBEDTLS_CAMELLIA_C and MBEDTLS_DES_C.
Concerning cipher modes MBEDTLS_CIPHER_MODE_WITH_PADDING has to be changed in psa_crypto.c.

Reply:
Ok, I'll make the changes in psa_crypto.c for the new defines.